File: PHP/Disk.php
<?php

/* membuat class */ 
class Disk{

/* deklarasi atribut private */
	private $type = "";
	private $capacity = 0;
	private $price = 0;
	
	/* constructor */
	public function __construct($type = 0, $capacity = 0, $price = 0){
		$this->type = $type;
		$this->capacity = $capacity;
		$this->price = $price;
	}

	/* setter dan getter untuk atribut type */
	public function setType($type){ 
		$this->type = $type;
	}

	public function getType(){
		return $this->type;
	}

	/* setter dan getter untuk atribut capacity */
	public function setCapacity($capacity){ 
		$this->capacity = $capacity;
	}

	public function getCapacity(){
		return $this->capacity;
	}

	/* setter dan getter untuk atribut price */
	public function setPrice($price){ 
		$this->price = $price;
	}

	public function getPrice(){
		return $this->price;
	}

	/* method untuk mencetak output atribut dari class */ 
	public function printDisk(){
		echo "Jenis Disk  : " . $this->type . " " . $this->capacity . " GB" . "<br>";
		echo "Harga       : Rp" . $this->price . "<br>";
		echo "------------------------------------------------------<br>";
	}
}

?>

File: C++/Disk.cpp
#include <iostream>

using namespace std;

/* membuat class */ 
class Disk{

/* deklarasi atribut private */
private:
	string type;
	int capacity;
	int price;

/* deklarasi method public */
public:
	
	/* constructor */
	Disk(){}

	Disk(string type, int capacity, int price){
		this->type = type;
		this->capacity = capacity;
		this->price = price;
	}

	/* setter dan getter untuk atribut type */
	void setType(string type){ 
		this->type = type;
	}

	string getType(){
		return this->type;
	}

	/* setter dan getter untuk atribut capacity */
	void setCapacity(int capacity){ 
		this->capacity = capacity;
	}

	int getCapacity(){
		return this->capacity;
	}

	/* setter dan getter untuk atribut price */
	void setPrice(int price){ 
		this->price = price;
	}

	int getPrice(){
		return this->price;
	}

	/* method untuk mencetak output atribut dari class */ 
	void printDisk(){
		cout << "Jenis Disk  : " << this->type << " " << this->capacity << " GB" << endl;
		cout << "Harga       : Rp" << this->price << endl;
		cout << "---------------------------------------" << endl;
	}

	/* destructor */
	~Disk(){}
};